mu/interface.py

```diff
@@ -755,6 +760,30 @@ def __init__(self, port, theme='day', parent=None):
             raise IOError("Cannot connect to device on port {}".format(port))
         self.set_theme(theme)
 
+    def repl_paste(self):
+        """
+        Grabs clipboard contents then sends down the serial port
+        """
+        clipboard = QApplication.clipboard()
+        self.serial.write(bytes(clipboard.text(), 'utf8'))
```

Comment:
clipboard.text() casts the clipboard contents to text, so it works for non-text objects. Empty string is fine (just writes nothing); None will break, so fixed in latest commit